src/Sankey.js

```diff
@@ -0,0 +1,233 @@
+/**
+    @external Viz
+    @see https://github.com/d3plus/d3plus-viz#Viz
+*/
+import {nest} from "d3-collection";
+import {sankey, sankeyLinkHorizontal} from "d3-sankey";
+
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import {Path} from "d3plus-shape";
+import * as shapes from "d3plus-shape";
+import {dataLoad as load, Viz} from "d3plus-viz";
+
+/**
+    @class Sankey
+    @extends external:Viz
+    @desc Creates a sankey visualization based on a defined set of XXXXX. [Click here](xxxxxx) for help getting started using the Sankey class.
+*/
+export default class Sankey extends Viz {
+
+  /**
+      @memberof Sankey
+      @desc Invoked when creating a new class instance, and sets any default parameters.
+      @private
+  */
+  constructor() {
+    super();
+    this._nodeId = accessor("id");
+    this._links = accessor("links");
+    this._noDataMessage = false;
+    this._nodes = accessor("nodes");
+    this._nodeWidth = 30;
+    this._on.mouseenter = () => {};
+    this._on["mouseleave.shape"] = () => {
+      this.hover(false);
+    };
```

Comment:
take a look at the Rings mousemove and mouseleave events, which highlight links based on the hovered shape: https://github.com/d3plus/d3plus-network/blob/master/src/Rings.js#L35-L67